test/src/main/webapp/META-INF/context.xml

```diff
@@ -180,5 +180,13 @@
 		factory="org.apache.naming.factory.BeanFactory"
 		type="org.apache.activemq.ActiveMQXAConnectionFactory"
 		brokerURL="tcp://host.docker.internal:61616"
-	/>	
+	/>
+
+	<!-- Activemq-artemis resource configuration -->
+	<Resource name="jms/qcf-activemq-artemis"
+		factory="org.apache.naming.factory.BeanFactory"
+		type="org.apache.activemq.artemis.jms.client.ActiveMQXAConnectionFactory"
+		brokerURL="tcp://host.docker.internal:61615"
```

Comment:
Here you use port 61615, while you published port 61616. I don't think that is close enough!